common/DatabaseController.js

```diff
@@ -0,0 +1,106 @@
+const { collection, doc, getDoc, getDocs, query, where } = require('firebase/firestore');
+const firebase = require('firebase/app');
+const moment = require('moment');
+const firebaseApp =require("firebase/app");
+const firestore = require ("firebase/firestore");
+
+
+class DatabaseController {
```

Comment:
There are errors we should handle from this class
Like connection errors with the DB, or things like that

Reply:
Can be considered in a testing branch, right?

Reply:
Yess